### ISSUE TITLE
plugin doesn't support Kibana 6.5.1

### ISSUE THREAD

**Issue by RachidKraiem** [nreese/kibana-time-plugin#45]
I'm facing problems to use plugin in latest 6.5.1 Kibana release
`Version: 6.5.1
Build: 18763
Error: Error: [$injector:unpr] Unknown provider: timefilterProvider <- timefilter <- KbnTimeVisController
http://errors.angularjs.org/1.6.9/$injector/unpr?p0=timefilterProvider%20%3C-%20timefilter%20%3C-%20KbnTimeVisController (http://localhost:5601/bundles/vendors.bundle.js:277)
window.onerror@http://localhost:5601/bundles/commons.bundle.js:3:866767`

Could someone look closer on that?

**Comment by RachidKraiem** [comment]
Hello,

Any help would be appreciated.

Regards,

**Comment by JoshuaSmeda** [comment]
Any support for 6.5.1 or plans to?

**Comment by nreese** [comment]
I just tried the plugin and it works in 6.5. What branch are you using? Master branch supports 6.5.

I added 6.5 instructions to the readme.

**Comment by RachidKraiem** [comment]
It works fine when using master branch.
Thanks !

**Comment by ZhenTu** [comment]
hi Nreese, we are integrating this time plugin with Kibana v6.5.4, however, we always encounter below error message when starting Kinaba after install this time plugin.
"kibana[41649]: Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`".
Do you have some idea about this? 
Thanks!

**Comment by nreese** [comment]
What steps did you take to get that error?

**Comment by ZhenTu** [comment]
> What steps did you take to get that error?

Hi Nreese, thank you for your reply! actually we are running Kibana on CentOS7.6 system, and there was no nodejs installed by default. In order to run the bower command, I install nodejs follow the commands in below link, https://www.rosehosting.com/blog/how-to-install-nodejs-bower-and-gulp-on-a-centos-7-vps, then I run the 'bower install' command and set kibana version to '6.5.4' in the package.json file.
Now I run the 'rpm install caniuse-lite browserslist -g' command, and get the latest node_modules and put them into kibana's node_module folder, and do not see the 'caniuse-lite is outdated' error again. however, I still can not get my kibana started properly. still not sure what's going wrong. This time, I see this log, {"type":"log","@timestamp":"2019-02-11T15:39:45Z","tags":["debug","legacy","proxy"],"pid":67306,"message":"Event is being forwarded: clientError"}.
By the way, the Kibana was working well before I install the time-plugin.

**Comment by nreese** [comment]
Just follow these instructions. You do not need to manually put anything in Kibana/node_modules.

```
cd KIBANA_HOME/plugins
git clone git@github.com:nreese/kibana-time-plugin.git (or git clone https://github.com/nreese/kibana-time-plugin.git)
cd KIBANA_HOME/plugins/kibana-time-plugin

git checkout master

bower install
vi kibana-time-plugin/package.json //set version to match kibana version
```

**Comment by ZhenTu** [comment]
yes, I first tried to follow this instruction. But it can not find bower command if I do not install nodejs and bower first.
